Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -428,6 +426,8 @@ void RenderGrid::layoutGrid(RelayoutChildren relayoutChildren)
         // It doesn't get included in the normal layout process but is instead skipped.
         layoutExcludedChildren(relayoutChildren);
 
+        auto aspectRatioBlockSizeDependentGridItems = computeAspectRatioDependentAndBaselineItems(gridLayoutState);
+
```

Comment:
This is error prone as lines tend to move around and it's easy to accidentally reverse their order since there's no clear indication of dependencies.